lib/helpers.js

```diff
@@ -24,4 +28,56 @@ async function getDirectories (rootPath) {
   return dirs.sort();
 }
 
-export { getDirectories };
+async function getAndroidPlatformAndPath () {
+  const androidHome = process.env.ANDROID_HOME;
+  if (typeof androidHome !== "string") {
+    log.error("ANDROID_HOME was not exported!");
+    return null;
+  }
+  // TODO import it from appium support
+  let locs = ['android-4.2', 'android-17', 'android-4.3', 'android-18',
+      'android-4.4', 'android-19', 'android-L', 'android-20', 'android-5.0',
+      'android-21'];
+
+  for(let loc of locs.reverse()) {
+    let platforms = path.resolve(androidHome, 'platforms');
```

Comment:
since you're doing `path.resolve(platforms, loc)` twice, I'd put it in a variable